Update documentation for packages built using python wheels

matt-chan opened the pull request:
The tests aren't included on packages build in PyPi. Updating documentation to show that.

sa2ajj commented on master/docs/manual/installation/installation.rst:
```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the buildbot unit test suite. You must not be using a python wheels packaged version 
```
Buildbot is spelled as `Buildbot`.

We have "one sentence per line policy" (http://docs.buildbot.net/latest/developer/style.html)

matt-chan commented:
Sorry about that. I didn't realize about the one-line thing and the capitalization. Fixed to reflect that now.

Also rebased to follow master again.

sa2ajj commented on master/docs/manual/installation/installation.rst:
```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the Buildbot unit test suite. 
+You must not be using a python wheels packaged version of Buildbot or have specified the bdist_wheel flag when building. 
```
Python is spelled Python.

sa2ajj replied:
I believe `bdist_wheel` is a command, not a flag.

I think an example of the command line with `bdist_wheel` would be great.

sa2ajj commented:
Otherwise, LGTM

matt-chan commented:
Rebased to follow master. Also applied changes discussed. Thanks!

sa2ajj commented:
Looks like it's still can't be merged to `master`.  Could you please rebase it once more?
